scripts/ingests/bones_archive/ingest_bones_sources.py

```diff
@@ -0,0 +1,163 @@
+from simple.schema import *
+from simple.schema import REFERENCE_TABLES
+from astrodb_utils import (
+    load_astrodb,
+    AstroDBError,
+)
+from astrodb_utils.sources import ingest_names, ingest_source, find_source_in_db
+from astrodb_utils.publications import ingest_publication, find_publication
+
+import sys
+from astrodb_utils.utils import logger
+sys.path.append(".")
+from astropy.io import ascii
+
+# logger = logging.getLogger(__name__)
+names_ingested = 0
+sources_ingested = 0
+skipped = 0
+total = 0
+already_exists = 0
+multiple_sources = 0
+no_sources = 0
+inside_if = 0
+
+
+DB_SAVE = True
+RECREATE_DB = False
+db = load_astrodb("SIMPLE.sqlite", recreatedb=RECREATE_DB, reference_tables=REFERENCE_TABLES)
+
+
+ingest_publication(db, doi="10.1088/0004-637X/748/2/93")  # Roja12
+ingest_publication(db, bibcode="2018MNRAS.479.1383Z", reference="Zhan18.1352")
+ingest_publication(db, bibcode="2018MNRAS.480.5447Z", reference="Zhan18.2054")
+
+ingest_source(
+    db,
+    "LHS 292",
+    search_db=False,
+    reference="Roja12",
+    ra_col_name="ra",
+    dec_col_name="deg",
+    epoch_col_name="epoch",
+)
```

Comment:
I've ingested LHS 292 directly and opened an issue with the bug in ingest_sources. https://github.com/astrodbtoolkit/astrodb_utils/issues/110